app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    results = {}
+
+    def inner(*args):
+        results
+        if args not in results.keys():
```

Comment:
```suggestion
        if args not in results:
```
It's a more clear way to check the key presence in the dictionary